docs/GroupDefs.hpp

```diff
@@ -1130,6 +1130,12 @@ then typically look as follows:
  * Things related to surfaces.
  */
 
+/*!
+ * \defgroup SwshGroup SWSH
```

Comment:
I think we should spell out what SWSH is in the doxygen group name (the second thing). I don't expect everyone to be familiar with it and I think it's better to be able to read the name and say "Nope, don't know so doesn't matter to me" than to have to guess and investigate.